### PR TITLE
FTS code cleanup

### PR DESCRIPTION
**Pull request by fangzheng**
Remove unused variables and macro, and fix typos in FTS code.
Reword notes in src/backend/fts/README.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Review by JunfengYang — COMMENTED** [review]
Nice job!

File: src/backend/fts/README [hunk 1]
```diff
@@ -6,10 +6,9 @@ This document illustrates the mechanism of a GPDB component called
 Fault Tolerance Service (FTS):
 
  - This sections explains how FTS probe process is started. The FTS
-   probe process is running on the Gp_entry_postmaster (master node)
-   only. It starts as a background worker process managed by the
-   BackgroundWorker structure. (see
-   src/include/postmaster/bgworker.h).  Greenplum sets up a group of
+   probe process is running on the master node only. It starts as a
```
[the remark this answers is not in the document]
Comment (fangzheng) [reply src/backend/fts/README]: Changed to 'coordinator'.

File: src/backend/fts/README [hunk 2]
```diff
@@ -22,11 +21,13 @@ Fault Tolerance Service (FTS):
...
 
-   In Postmaster, we will check the following condition:
+   FtsProbeStartRule() function specifies the following condition under
+   which the FTS probe process should be started by postmaster:
 
-      Gp_entry_postmaster && Gp_role == GP_ROLE_DISPATCH
+      Gp_role == GP_ROLE_DISPATCH
 
-   The FTS probe process is started when the condition is true.
+   That is, the FTS probe process should only be started on the master
```
[the remark this answers is not in the document]
Comment (fangzheng) [reply src/backend/fts/README]: Changed to 'coordinator'.

**Comment by fangzheng** [comment]
> Nice job!

Hi Junfeng, thanks for the review comments. I've made a push to change the term 'master' to 'coordinator' in the document.

**Review by gfphoenix78 — APPROVED** [review]

**Pull request merged**